go.mod

```diff
@@ -1,34 +1,66 @@
 module github.com/banzaicloud/nodepool-labels-operator
 
-go 1.13
+go 1.17
```

Comment:
**Question**: Is there any reason not to use 1.18 here?

Reply:
No particular reason, can do 1.18 as well! 🙂